UnityProject/Assets/Scripts/Objects/Alien/AlienEggCycle.cs

```diff
@@ -189,6 +213,11 @@ private void FeelsSlimy(Interaction interaction)
 				$"{interaction.Performer.ExpensiveName()} touches the slimy egg!");
 		}
 
+		public bool WillInteract(HandApply interaction, NetworkSide side)
+		{
+			return DefaultWillInteract.Default(interaction, side) &&
+				!(interaction.Intent == Intent.Harm && currentState == EggState.Grown);
```

Comment:
This is gonna be an issue since currentState isnt sync'd

Reply:
I'm sorry I'm new to this and I though OnSpawnServer() is somehow syncing it, so how am I supposed to sync it?

Reply:
`
if(DefaultWillInteract.Default(interaction, side) == false) return false;

return side == NetworkSide.Server ? !(interaction.Intent == Intent.Harm && currentState == EggState.Grown) : true;
`

Think that'll be better than having to sync the state

Reply:
Thanks for help, I used your approach in my last commit